### PR TITLE
Login and sign up Ui

### PR DESCRIPTION
**Pull request by dhgavali**
* login & Sign up UI Completed
* form validation done

### CODE REVIEW

File: lib/view/authentication/login_screen.dart [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+              Column(
+                children: [
+                  Container(
+                    padding: EdgeInsets.symmetric(
+                      horizontal: 0,
+                      vertical: 35,
+                    ),
+                    decoration: BoxDecoration(
+                      borderRadius: BorderRadius.circular(8),
+                      color: Color(0xffffffff).withOpacity(0.05),
```
Comment (PranavMasekar) [inline lib/view/authentication/login_screen.dart]: User AppColors class to user colors in ui

File: lib/view/authentication/login_screen.dart [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+                                  "Forgot Password ?",
+                                  style: MyTextStyles.poppins500,
+                                )),
+                          ),
+                          PrimaryBtn(
+                            label: "Login",
+                            onpress: () {
+                              //  TODO: Login Function call
+                              if (_formkey.currentState!.validate()) {
+                                ScaffoldMessenger.of(context).showSnackBar(
```
Comment (PranavMasekar) [inline lib/view/authentication/login_screen.dart]: Use customSnackbar at time of API integration

File: lib/view/authentication/login_screen.dart [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+            ),
+          ),
+          Expanded(child: Divider(color: Colors.white, thickness: 2)),
+        ],
+      ),
+    );
+  }
+}
+
+class HeadLine extends StatelessWidget {
```
Comment (PranavMasekar) [inline lib/view/authentication/login_screen.dart]: Export entire logo from figma , Don't need to create our own logo headline

File: lib/view/authentication/login_screen.dart [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+            "assets/images/github.png",
+            width: 40,
+          ),
+        ),
+      ],
+    );
+  }
+}
+
+class OrLine extends StatelessWidget {
```
Comment (PranavMasekar) [inline lib/view/authentication/login_screen.dart]: Move this to separate file under widgets folder

**Review by PranavMasekar — CHANGES_REQUESTED** [review]

**Comment by dhgavali** [comment]
Changes done

File: lib/view/authentication/login_screen.dart (hunk 1 above)
Comment (dhgavali) [reply lib/view/authentication/login_screen.dart]: Added colors from AppColors class

File: lib/view/authentication/login_screen.dart (hunk 2 above)
Comment (dhgavali) [reply lib/view/authentication/login_screen.dart]: removed the snackbar.

File: lib/view/authentication/login_screen.dart (hunk 3 above)
Comment (dhgavali) [reply lib/view/authentication/login_screen.dart]: Done

File: lib/view/authentication/login_screen.dart (hunk 4 above)
Comment (dhgavali) [reply lib/view/authentication/login_screen.dart]: exported to widgets/login_widgets.dart

**Review by PranavMasekar — APPROVED** [review]

**Pull request merged**